docs/schulcloud-server/Coding-Guidelines/controllers.md

```diff
@@ -0,0 +1,49 @@
+# Controller
+
+A modules api layer is defined within of [controllers](https://docs.nestjs.com/controllers).
+
+The main responsibilities of a controller is to define the REST API interface as openAPI specification and map DTO's to match the logic layers interfaces.
+
+```TypeScript
+    @Post()
```

Comment:
Make it sense to also put a example line for expected errors open API definition.
Empty line between 15-16